spec/use_cases/administrator/get_auth_requests_spec.rb

```diff
@@ -0,0 +1,51 @@
+describe UseCases::Administrator::GetAuthRequests do
+  let(:authentication_logs_gateway) { double }
+  let(:search_validator) { double(execute: { success: true }) }
+
+  subject do
+    described_class.new(
+      authentication_logs_gateway: authentication_logs_gateway,
+      search_validator: search_validator
+    )
+  end
+
+  before do
+    allow(authentication_logs_gateway).to receive(:search)
```

Comment:
I think this line isn't needed - RSpec doubles are permissive by default. Just testing it out locally.

Reply:
Okay, they're _strict_ by default, so I'm _100% wrong_.

I would still not use this, I would put it on the spy declaration above:
```ruby
let(:authentication_logs_gateway) { spy(search: nil) }
```

(did test this out now, and it works)

Reply:
`instance_spy` of that gateway would be even more strict.